user/how-to-guides/backup-emergency-restore-v3.md

```diff
@@ -64,7 +66,7 @@ any GNU/Linux system with the following procedure.
     supported message digest algorithms can be found with `openssl
     list-message-digest-algorithms`.
 
- 4. Read the `backup-header`. You'll need some of this information later. The
+ 5. Read the `backup-header`. You'll need some of this information later. The
```

Comment:
Previous point is "3", so why "5" here (and renumbering others below)?

Reply:
That was not my intention. I can correct that. Thanks for pointing out!